_sass/config/_lazy.scss

```diff
@@ -6,10 +6,14 @@ picture img {
   width: 100%; /* stretch to fill the picture element */
 }
 
-img.lazyloaded {
+img.lazyloaded,
+.panel--banner.lazyloaded,
+.img-full-width.lazyloaded {
```

Comment:
this is specified like so because I didn't want to apply the filter to the hero section on the homepage, since it blurs the text initially too.